docs/source/installation/osx.rst

```diff
@@ -24,28 +22,28 @@ Python package
 
 2. Install Powerline using one of the following commands:
 
+    This will get the current release version:
```

Comment:
This line looks clumsy with the preceding line. Also indentation is wrong: should be three spaces. Previous variant that formed a sentence with code blocks looks better.